src/gridstack.js

```diff
@@ -296,7 +296,8 @@
 
   var idSeq = 0;
 
-  var GridStackEngine = function(column, onchange, float, maxRow, items) {
+  var GridStackEngine = function(opts, column, onchange, float, maxRow, items) {
```

Comment:
so column, float and maxRow are all grid options so it doesn't make sense to pass generic options there (for rtl) but others spelled out, You are breaking API anyway so might as well clean it up.

Reply:
Options object used for most params now. Signature is now GridStackEngine(opts, onChange, nodes)